addons/survey/security/survey_security.xml

```diff
@@ -1,24 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data noupdate="1">
-        <!-- Survey users -->
-        <record model="res.groups" id="base.group_survey_user">
-            <field name="name">Survey / User</field>
-        </record>
-
-        <!-- Survey managers -->
-        <record model="res.groups" id="base.group_survey_manager">
-            <field name="name">Survey / Manager</field>
-            <field name="implied_ids" eval="[(4, ref('base.group_survey_user'))]"/>
-            <field name="users" eval="[(4, ref('base.user_root'))]"/>
-        </record>
 
         <!-- Record rules -->
         <record id="survey_users_access" model="ir.rule">
             <field name="name">Access to survey for regular users</field>
             <field name="model_id" ref="survey.model_survey_survey"/>
             <field name="domain_force">[('stage_id.closed', '=', False)]</field>
-            <field name="groups" eval="[(4, ref('base.group_survey_user'))]"/>
+            <field name="groups" eval="[(4, ref('base.group_marketing_user'))]"/>
```

Comment:
Incorrect, anyone can answer a survey, even people that do not use marketing module at all